pinot-core/src/main/java/org/apache/pinot/core/util/ListenerConfigUtil.java

```diff
@@ -143,6 +143,27 @@ private ListenerConfigUtil() {
     return listeners;
   }
 
+  public static List<ListenerConfig> buildMinionAdminConfigs(PinotConfiguration minionConf) {
+    List<ListenerConfig> listeners = new ArrayList<>();
+
+    String adminApiPortString = minionConf.getProperty(CommonConstants.Minion.CONFIG_OF_ADMIN_API_PORT);
+    if (adminApiPortString != null) {
```

Comment:
Is this default behavior necessary at all? We could simply require this config to be provided or not bring up the minion admin api at all

Reply:
This is what all other components are doing as well, I tried to match that.